pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc

```diff
@@ -26,6 +28,8 @@ To create a custom policy project, run the PDK CLI `create` command, replacing `
 anypoint-cli-v4 pdk policy-project create --name <my-custom-policy>
 ----
 
+NOTE: If you receive a prompt requesting your group ID, see xref:policies-pdk-troubleshooting.adoc#group-id-error[].
```

Comment:
In the past, if you linked to an anchor like this, you'd need to include the title in the []. Is this no longer the case?